3.resistance-signature/results/singscore/UNIQUE_README.md

```diff
@@ -0,0 +1,6 @@
+UNIQUE_README.md
+
+Examination of files revealed some line duplications. Files labeled as UNIQUE had
+line duplications in the original file and these duplications were removed in the
+UNIQUE file. UNIQUE files were used for figure generation whenever these errors
+were detected.
```

Comment:
```suggestion
Examination of files revealed some line duplications.
Files labeled as UNIQUE had line duplications in the original file and these duplications were removed in the UNIQUE file.
UNIQUE files were used for figure generation whenever these errors were detected.
```

Please do this for the other two READMEs

Reply:
READMEs are reformatted and volcanos.pzfx was uploaded using LFS

Not sure which data/result changes were concerning. I've added csv files since it was easier for me to review and pull data from them vs. tsv, is this what you're referring to? The UNIQUE files are new but their generation was described in the UNIQUE_README

Reply:
The only reason I added them to the repo is that the code I use calls for them (couldn't explore tsv files) and I didn't want there to be any confusion. Also, those labeled UNIQUE have the duplicate values removed and are therefore not identical to their tsv counterparts

Reply:
got it. My vote is to keep the UNIQUE files, remove the `csv` files, and add a note in the code that says you used a csv converted files.

Doing so 1) reduces bloat in the repo and 2) maintains a connection between the `tsv` files (which may change in future versions)

What do you think?

Reply:
That's fine with me. I'll update the relevant code with comments re:CSV files. Is it straightforward to selectively approve the pull request to only add the UNIQUE files to the repo?

Reply:
you'll have to remove the non-UNIQUE labeled csv files with `git rm PATH/TO/CSVFILES.csv`

Reply:
I still see 6 files in the `3.resistance-signature/data/` that are not labeled UNIQUE. Can you remove changes to these files? I don't think they should be updated in this PR, but I could be wrong! If they should be updated, can you provide justification?

(it's super important for them to not be updated unless necessary, to retain reproducibility)